Cargo.toml

```diff
@@ -13,21 +13,18 @@ doctest = false
 [features]
 unstable = [
     "string_cache/unstable",
-    "tendril/unstable",
     "rc/unstable",
     "html5ever/unstable",
     "selectors/unstable",
 ]
-with-hyper = ["hyper"]
```

Comment:
It was moved to https://github.com/servo/html5ever/commit/f470e3eb6fda97d9935aab3d5489ce2881522430. This line should be replaced with `with-hyper = ["html5ever/with-hyper"]` but that won’t work until a new version of html5ever with that feature is on crates.io.